src/countCakes.test.js

```diff
@@ -8,13 +8,109 @@ describe(`Function 'countCakes':`, () => {
   });
 
   it(`should return a number`, () => {
+    const res = countCakes();
 
+    expect(typeof res).toBe('number');
   });
 
   it(`should return the number of cakes 
     when Pete has as many ingredients as the recipe requires`, () => {
+    const recepi = {
+      flour: 500,
+      sugar: 200,
+      eggs: 5,
+    };
+    const available = {
+      flour: 500,
+      sugar: 200,
+      eggs: 5,
+    };
 
+    expect(countCakes(recepi, available)).toBe(1);
```

Comment:
My suggestion is to add to text the result number (for example 'should return the number of cakes 1 ...)